### ISSUE TITLE
IRC-style message layout should have usernames right-justified

### ISSUE THREAD

**Issue by jasondunsmore** [vector-im/element-web#24413]
### Your use case

When IRC-style message layout is enabled, it's difficult to visually match usernames to messages when you have some long usernames and some short usernames chatting in the same room. Dragging the divider helps a little, but then some usernames are truncated.

The usernames should be right-justified like they are in Hexchat:
https://i.imgur.com/tLMguQz.png

**Comment by gaelledel** [comment]
@amshakal To be discussed as a team

**Comment by gaelledel** [comment]
Agreed to have the usernames  right-justified instead to tackle this issue. @MidhunSureshR Could you check if this would be a good candidate to tackle as part of the maintainer tasks? Thanks
